lib/elixir/test/elixir/macro_test.exs

```diff
@@ -803,7 +803,8 @@ defmodule MacroTest do
               true
             else
               :error -> false
-            end
+            end,
+            print_location: false
```

Comment:
It uses the dbg function and prints the header now  before it crashes so it has to be muted.
Before any crash was silent.